sentry/table_sentry_metric_alert.go

```diff
@@ -0,0 +1,200 @@
+package sentry
+
+import (
+	"context"
+
+	"github.com/jianyuan/go-sentry/v2/sentry"
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+func tableSentryMetricAlert(ctx context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "sentry_metric_alert",
+		Description: "Retrieve information about your metric alerts.",
+		List: &plugin.ListConfig{
+			ParentHydrate: listProjects,
+			Hydrate:       listMetricAlerts,
+			KeyColumns: []*plugin.KeyColumn{
+				{
+					Name:    "project_slug",
+					Require: plugin.Optional,
+				},
+			},
+		},
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.AllColumns([]string{"organization_slug", "project_slug", "id"}),
+			Hydrate:    getMetricAlert,
+		},
+		Columns: []*plugin.Column{
+			{
+				Name:        "id",
+				Type:        proto.ColumnType_STRING,
+				Description: "The ID of this metric alert.",
+				Transform:   transform.FromField("ID"),
+			},
+			{
+				Name:        "name",
+				Type:        proto.ColumnType_STRING,
+				Description: "The metric alert name.",
+			},
+			{
+				Name:        "owner",
+				Type:        proto.ColumnType_STRING,
+				Description: "Specifies the owner id of this metric alert rule.",
+			},
+			{
+				Name:        "organization_slug",
+				Type:        proto.ColumnType_STRING,
+				Description: "The slug of the organization the metric alert belongs to.",
+			},
+			{
+				Name:        "project_slug",
+				Type:        proto.ColumnType_STRING,
+				Description: "The slug of the project the metric alert belongs to.",
+			},
+			{
+				Name:        "aggregate",
+				Type:        proto.ColumnType_STRING,
+				Description: "The aggregation criteria to apply.",
+			},
+			{
+				Name:        "data_set",
+				Type:        proto.ColumnType_STRING,
+				Description: "The Sentry metric alert category.",
+			},
+			{
+				Name:        "date_created",
+				Type:        proto.ColumnType_TIMESTAMP,
+				Description: "The creation timestamp of the metric alert.",
+			},
+			{
+				Name:        "environment",
+				Type:        proto.ColumnType_STRING,
+				Description: "Perform metric alert rule in a specific environment.",
+			},
+			{
+				Name:        "query",
+				Type:        proto.ColumnType_STRING,
+				Description: "The query filter to apply.",
+			},
+			{
+				Name:        "resolve_threshold",
+				Type:        proto.ColumnType_DOUBLE,
+				Description: "The value at which the metric alert rule resolves.",
+			},
+			{
+				Name:        "task_uuid",
+				Type:        proto.ColumnType_STRING,
+				Description: "The UUID of the async task that can be spawned to create the metric alert.",
+				Transform:   transform.FromField("TaskUUID"),
+			},
+			{
+				Name:        "threshold_type",
+				Type:        proto.ColumnType_INT,
```

Comment:
Is the data type of the column correct?

Reply:
yes, https://pkg.go.dev/github.com/jianyuan/go-sentry/v2/sentry#MetricAlert.ThresholdType